tests/shadow/test_shadow_measurement.py

```diff
@@ -25,13 +26,7 @@ def get_circuit(wires, shots, seed_recipes, interface="autograd", device="defaul
     Return a QNode that prepares the state (|00...0> + |11...1>) / sqrt(2)
         and performs the classical shadow measurement
     """
-    if device is not None:
-        dev = qml.device(device, wires=wires, shots=shots)
-    else:
-        dev = qml.device("default.qubit", wires=wires, shots=shots)
-
-        # make the device call the superclass method to switch between the general qubit device and device specific implementations (i.e. for default qubit)
-        dev.classical_shadow = super(type(dev), dev).classical_shadow
+    dev = qml.device(device, wires=wires, shots=shots)
```

Comment:
I think we still have to create a mock device without a classical shadow implementation, instead of relying on the fact that `default.mixed` doesn't have one.

Eventually we'll want to support a fast implementation of classical shadows for `default.mixed`, just like we have for `default.qubit` currently. In this case, the default implementation of classical shadows in `ClassicalShadow.process()` won't be tested and we'll lose coverage.

Reply:
Makes sense. Changed back to original.